### PR TITLE
Removing DS Date.

### PR DESCRIPTION
**Pull request by nibivi77**
Adding our own Date component.

### CODE REVIEW

File: components/Date.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+export function Date(props) {
+  const { id, label, date } = props
+  const dateFormatted = date ? date.replace(/^(.{4})(.{2})/gm, '$1-$2-') : 'NA'
+  return (
+    <dl id={id} data-testid={id} className="ds-mt-8 ds-py-2 ds-font-body">
```
Comment (shewood) [inline components/Date.js:8]: You will need to remove the ds prefix here for the classes

Comment (nibivi77) [reply components/Date.js:8]: Good catch

File: components/Date.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types'
+
+export function Date(props) {
+  const { id, label, date } = props
+  console.log(date)
```
Comment (krischarbonneau) [inline components/Date.js:5]: Stray console log :)

**Review by krischarbonneau — APPROVED** [review]
Looks good!

**Pull request merged**